<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign in : </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_login">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_register">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuThe_Best_Chat_App_Ever">
    <property name="title">
     <string>The Best Chat App Ever</string>
    </property>
   </widget>
   <addaction name="menuThe_Best_Chat_App_Ever"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
